### PR TITLE
[EDU-3118] feat: add secure infrastructure

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added new page on Secure an infrastructure.

Note: Some links aren't working yet.

[EDU-3118]: https://aziontech.atlassian.net/browse/EDU-3118?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/secure-journey/secure-infrastructure.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+namespace: docs_secure_infrastructure
+menu_namespace: secureMenu
+permalink: /documentation/products/secure/secure-infrastructure/
+---
+
+import Button from '~/components/Button.astro'
+
+Once you've [created an edge firewall and secured your application](/en/documentation/products/secure/secure-application/), you can use **Origin Shield** to secure your infrastructure by blocking threats.
+
+Origin Shield works as a list of IP/CIDR addresses used in Azion's Edge, thus providing ensurance of safe IPs to allow access. The list is maintained by Azion and available via Real-Time Manager (RTM) and API, but each client must create automations to guarantee their security policies are always up-to-date with the Origin Shield addresses.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/secure-infrastructure.mdx:14]:
```suggestion
Origin Shield works as a list of IP/CIDR addresses used in Azion's edge, thus providing assurance of safe IPs to allow access. The list is maintained by Azion and available via Real-Time Manager (RTM) and API, but each client must create automation to guarantee their security policies are always up-to-date with the Origin Shield addresses.
```

**Review by MarianaAguilera — APPROVED** [review]
just a suggestion.

File: src/content/docs/en/pages/secure-journey/secure-infrastructure.mdx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+## Step 1. Enable Origin Shield
+
+To access the Origin Shield list, you must have:
+
+- An [edge firewall created](/en/documentation/products/secure/secure-application/).
+- The [Network Layer Protection module](/en/documentation/products/guides/secure/configure-main-settings/#modules) enabled.
+
+---
+
+## Step 2. Access the list of addressess
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/secure-journey/secure-infrastructure.mdx:27]:
```suggestion
## Step 2. Access the list of addresses
```

File: src/content/docs/en/pages/secure-journey/secure-infrastructure.mdx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    }
+  ]
+}
+```
+
+3. On the `ip_list` field, you'll receive all IPs added by Azion. You can copy the IP addresses list.
+
+---
+
+## Step 3. Automate new addresses additions
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/secure-journey/secure-infrastructure.mdx:81]:
```suggestion
## Step 3. Automate new address additions
```

**Review by gabriel-azion — APPROVED** [review]

**Pull request merged**